core/spec/models/spree/wallet/add_payment_sources_to_wallet_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Spree::Wallet::AddPaymentSourcesToWallet, type: :model do
+  let(:order) { create(:order_ready_to_complete) }
+
+  describe '#add_to_wallet' do
+    subject { described_class.new(order) }
+
+    it 'saves the payment source and calls make_default' do
```

Comment:
This doesn't seem to actually test that the `make_default` method is called. I'm not sure whether we should do it at all tbh: on one end, it's an extension point and it's probably worth testing; on the other, it's a private interface and it's generally considered bad practice to test them. What do you think?